devices/Pcd8544/README.md

```diff
@@ -0,0 +1,178 @@
+# PCD8544 - 48 × 84 pixels matrix LCD, famous Nokia 5110 screen
+
+This is the famous screen that Nokia 5110 used. It's a SPI based device. This Nokia 5110 was very popular and many of us used to have it in their hands and that young generation have seen it in pictures. This LCD is quite cheap and easy to use.
+
+## Documentation
+
+- Nokia5110 [datasheet](https://www.sparkfun.com/datasheets/LCD/Monochrome/Nokia5110.pdf)
+
+## Usage
+
+The screen is most of the time presented like this:
+
+![Nokia 5110 screen](./Nokia-5110-LCD-Pinout.png)
```

Comment:
As this is nice that we normally do [Fritzing](https://fritzing.org/) schemas + export to png file.
You can find in the internet a free version of it or ping me on Discord ;)

Reply:
Done

![image](https://user-images.githubusercontent.com/8640943/188442771-b727308d-80c8-4264-b54b-7e89395eca12.png)